.github/workflows/changelog_reminder.yml

```
@@ -0,0 +1,24 @@
+on:
+  pull_request:
+    types: [opened, ready_for_review]
+name: Changelog Reminder
+jobs:
+  remind:
+    name: Changelog Reminder
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Changelog Reminder
+      uses: peterjgrainger/action-changelog-reminder@v1.3.0
+      with:
+        changelog_regex: '.github/CHANGELOG.md'
+        customPrMessage: |
+          Hello. You may have forgotten to update the changelog!
+          Please edit [.github/CHANGELOG.md](/PennyLaneAI/pennylane/blob/master/.github/CHANGELOG.md) with:
```

Would the complete URL be required here? The link that was posted as a comment doesn't seem to work.
```suggestion
          Please edit [.github/CHANGELOG.md](https://github.com/PennyLaneAI/pennylane/blob/master/.github/CHANGELOG.md) with:
```

---

I had the the link and text location flipped before. Works now.